Rally/app/src/main/java/com/example/compose/rally/ui/overview/OverviewScreen.kt

```diff
@@ -138,9 +142,11 @@ private fun AlertItem(message: String) {
         )
         IconButton(
             onClick = {},
-            modifier = Modifier.align(Alignment.Top)
+            modifier = Modifier
+                .align(Alignment.Top)
+                .clearAndSetSemantics {}
         ) {
-            Icon(Icons.Filled.Sort, contentDescription = stringResource(id = R.string.sort))
+            Icon(Icons.Filled.Sort, contentDescription = null)
```

Comment:
why wouldn't you want that?

Reply:
I was trying to figure out what this button would actually do when you press it. Seemed weird to have a "sort" button on each alert. 

Also; having an identical id for a button on each row in a column is considered a bad practice, as voice access users won't be able to identify them, and focusing each of them would be a repeated effort. In this case, I would use a custom action on the row instead, that describes the action the user can take.